front/src/applications/operationalStudies/views/Scenario.tsx

```diff
@@ -47,6 +49,8 @@ export default function Scenario() {
     osrdEditoastApi.endpoints.getProjectsByProjectIdStudiesAndStudyIdScenariosScenarioId.useLazyQuery(
       {}
     );
+  const [getTimetableByIdConflicts] =
```

Comment:
```suggestion
  const [getTimetableByIdConflicts: getTimetableConflict] =
```
You can rename the request here. Personally, I prefer to use query rather than LazyQuery, what about you ?

Reply:
Done ! :)